composables/browse.ts

```diff
@@ -5,3 +5,27 @@ export function useBrowse() {
     useLazyAsyncData("browse", () => new BrowseApi().browseGet()),
   );
 }
+
+export function useBrowseEvents() {
+  return reactiveApi(
+    useLazyAsyncData("browse", () => new BrowseApi().browseEventsGet()),
```

Comment:
If the result of this API call differs from `BrowseApi().browseGet()`, it should have another key - not `browse`. Otherwise the caching algorithm will most likely create some problems in the future 🙈

Reply:
copy paste error. done